### PR TITLE
Rearchitect two integration tests to use page objects

### PR DESCRIPTION
**Pull request by swalchemist**
This uses the page object pattern to improve two integration tests that use Selenium to drive Chrome. This could eventually be used to make all of the Selenium tests easier to maintain with less code duplication. This also abstracts most of the Selenium API away from the test cases.

### CODE REVIEW

**Review by peterhaochen47 — APPROVED** [review]

**Comment by Tallicia** [comment]
This introduces a paradigm that commits the project to years of additional refactoring to remain consistent. This change required more approval and discussion to understand the long term cost.

**Comment by swalchemist** [comment]
@Tallicia The PR was open for 10 days for feedback before it was merged.

But I'd welcome additional discussion, as I intend to make further changes using this pattern.

**Pull request merged**
